CHANGELOG.md

```diff
@@ -5,6 +5,193 @@ information on the list of deprecated flags and APIs please have a look at
+  - Improve root validation and yubikey debug logging [docker/notary#858](https://github.com/docker/notary/pull/858) [docker/notary#891](https://github.com/docker/notary/pull/891)
+  - Warn if certificates for root or delegations are near expiry [docker/notary#802](https://github.com/docker/notary/pull/802)
+  - Warn if role metadata is near expiry [docker/notary#786](https://github.com/docker/notary/pull/786)
+  - Fix passphrase retrieval attempt counting and terminal detection [docker/notary#906](https://github.com/docker/notary/pull/906)
+- Avoid unnecessary blob uploads when different users push same layers to authenticated registry [#26564](https://github.com/docker/docker/pull/26564)
+* Allow external storage for registry credentials [#26354](https://github.com/docker/docker/pull/26354)
+
+### Logging
+
+* Standardize the default logging tag value in all logging drivers [#22911](https://github.com/docker/docker/pull/22911)
+- Improve performance and memory use when logging of long log lines [#22982](https://github.com/docker/docker/pull/22982)
++ Enable syslog driver for windows [#25736](https://github.com/docker/docker/pull/25736)
++ Add Logentries Driver [#27471](https://github.com/docker/docker/pull/27471)
++ Update of AWS log driver to support tags [#27707](https://github.com/docker/docker/pull/27707)
++ Unix socket support for fluentd [#26088](https://github.com/docker/docker/pull/26088)
+* Enable fluentd logging driver on Windows [#28189](https://github.com/docker/docker/pull/28189)
+- Sanitize docker labels when used as journald field names [#23725](https://github.com/docker/docker/pull/23725)
+
+### Networking
+
++ Add `--attachable` network support to enable `docker run` to work in swarm-mode overlay network [#25962](https://github.com/docker/docker/pull/25962)
++ Add support for host port PublishMode in services using the `--port` option in `docker service create` [#27917](https://github.com/docker/docker/pull/27917)
++ Add support for Windows server 2016 overlay network driver (requires upcoming ws2016 update) [#28182](https://github.com/docker/docker/pull/28182)
+* Change the default `FORWARD` policy to `DROP` [#28257](https://github.com/docker/docker/pull/28257)
++ Add support for specifying static IP addresses for predefined network on windows [#22208](https://github.com/docker/docker/pull/22208)
+- Fix `--publish` flag on `docker run` not working with IPv6 addresses [#27860](https://github.com/docker/docker/pull/27860)
+- Fix inspect network show gateway with mask [#25564](https://github.com/docker/docker/pull/25564)
+- Fix an issue where multiple addresses in a bridge may cause `--fixed-cidr` to not have the correct addresses [#26659](https://github.com/docker/docker/pull/26659)
++ Add creation timestamp to `docker network inspect` [#26130](https://github.com/docker/docker/pull/26130)
+- Show peer nodes in `docker network inspect` for swarm overlay networks [#28078](https://github.com/docker/docker/pull/28078)
+
```

Comment:
- Enable ping for service VIP address [#28019] (https://github.com/docker/docker/pull/28019)